Libraries/LibWeb/Layout/GridFormattingContext.h

```diff
@@ -268,7 +268,7 @@ class GridFormattingContext final : public FormattingContext {
 
     AvailableSize get_free_space(AvailableSpace const&, GridDimension const) const;
 
-    Optional<int> get_line_index_by_line_name(GridDimension dimension, String const&);
+    Optional<int> get_line_index_by_line_name(GridDimension dimension, String const&, int line_number);
```

Comment:
`get_line_index_by_line_name` is only supposed to be used when you need to get index by line name, so adding `line_number` argument seems very confusing. we should not reach this function if we have line number.

Reply:
I agree. Would you be fine with renaming the argument to `explicit_line_number` or something similar? One could also rename the function to `get_line_index_by_line_name_and_explicit_line_number` to make the purpose clearer though that seems like a somewhat verbose name.

Reply:
The "issue" is that this function is now exclusively used in a context where the user-provided line number is relevant. I have renamed it to `get_nth_line_index_by_line_name`. How do you like this approach?